src/components/calendar/AssociatedClass.test.js

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { wrapperCreator, mockStyles } from 'util/testing';
 import * as timeUtils from 'util/time';
 import Section from 'components/common/Section';
 import ClassModal from './ClassModal';
-import { UnstyledAssociatedClass, styles, MAX_WIDTH_PERCENT } from './AssociatedClass';
+import AssociatedClass, { styles, MAX_WIDTH_PERCENT } from './AssociatedClass';
 
 jest.mock('util/time');
 
-describe('dynamic styles', () => {
-  const dow = 'Mo';
-  const event = {
-    dow,
-    start: {
-      hour: 10,
-      minute: 30,
-    },
-    end: {
-      hour: 12,
-      minute: 0,
-    },
-  };
-  const hour = 10;
-  const color = 'some color';
-  const associatedClass = { event, color, column: 0, columnWidth: 1 };
+const dow = 'Mo';
```

Comment:
Let's keep this inside the styles block just for grouping's sake

Reply:
Done